### ISSUE TITLE
"--exclude-if-present" flag for multiple files

### ISSUE THREAD

**Issue by FoxP** [ncw/rclone#2423]
#### What is the problem you are having with rclone?

"--exclude-if-present" flag only supports one filename, i.e. it can not be used multiple times.
Let's say i have a directory structure like this:

```
pictures/2017/Paris trip
pictures/2017/Barcelona trip
pictures/2017/New York trip
pictures/2018/Fun with friends
pictures/2018/Family trip
```

I would like to mount folders that contain a `.friend` file only on friends computers
I would like to mount folders that contain a `.family` file only on family computers
I would like to mount folders that contain a `.gf` file only on my girlfriend computer
Etc...

With the "--exclude-if-present" implementation, i can't do this kind of thing, ie for friends i can't do:

`--exclude-if-present .family .gf`

#### What is your rclone version (eg output from `rclone -V`)

Latest (1.42)

**Comment by ncw** [comment]
It would be relatively straight forward to let the `--exclude-if-present` flag be repeated or possibly turn its argument into a glob pattern.

Fancy helping to implement this?

**Comment by kwsp** [comment]
Currently, the flag can be trivially repeated by passing it multiple times: `--exclude-if-present .family --exclude-if-present .gf`. I think this is good enough. If we want to make the flag accept multiple arguments, we need to decide how to separate multiple arguments - space would be bad because how do we know when to stop parsing args for this flag? ',' (csv) could be bad because it's legal to have ',' in the filename. I vote to close this issue.

**Comment by ncw** [comment]
Yes we did this already in f4f0e444bf60b7bf6ebac9fa1e8a94d6821bdd40 as part of #6219 . Repeating the flag is how it is intended to work.

So I will close this!
